src/client/search/Current.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { addSearchTerm } from './SearchActions';
+import SearchTermsList from './SearchTermsList';
+
+let nextSearchTermId = 0;
+let Current = ({ dispatch }) => {
+	let showSearchkeyword = false;
+	let searchKeyword;
+	let searchKeywordContainer;
+	return(
+		<div className="row ui raised segment">
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2183766983/issues/source?bid=3148782&fileBranchId=3165072#l12)